src/data/nav-items.yaml

```diff
@@ -201,8 +201,8 @@
   pages:
     - title: Overview
       path: /community/overview
-    - title: Components
-      path: /community/components
+    - title: Component Index
```

Comment:
Do we need to remove the components page itself? https://github.com/carbon-design-system/carbon-website/blob/master/src/pages/community/components.mdx